README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
> Although it looks simple, that would be very ineffective. We still lack the coverage for important features of ES6 today and a tag from the day it was published would not give a reasonable value to use it.

That's fine. There's no value in the old versions anyway.

> a filter using the frontmatter tags is still more effective regarding what we still have today.

Effective for whom? Literally zero consumers of test262 are asking for this.